### PR TITLE
Add Gabor Somogyi to committers

### PR DESCRIPTION
**Pull request by gaborgsomogyi**

### CODE REVIEW

**Comment by gaborgsomogyi** [comment]
cc @HeartSaVioR

File: committers.md [hunk 1]
```diff
@@ -53,6 +53,7 @@ navigation:
 |Cheng Lian|Databricks|
 |Yanbo Liang|Facebook|
 |Jungtaek Lim|Cloudera|
+|Gabor Somogyi|Cloudera|
```
Comment (HeartSaVioR) [inline committers.md]: Actually there's a convention in the list if I understand correctly (I don't know where this is documented) - the list is alphabetically sorted by "last name" (not first name).

Could you please check the sequence a bit and please update the same if the list follows the pattern?

**Review by HeartSaVioR — COMMENTED** [review]
Btw, congrats! You deserve it!

File: committers.md (hunk 1 above)
Comment (gaborgsomogyi) [reply committers.md]: Oh, not seen pattern in the first letter so thought the order doesn't matter, checking...

**Review by HeartSaVioR — APPROVED** [review]
+1 you could now try to push to the repo. Congrats!

**Review by HyukjinKwon — APPROVED** [review]
Congrats @gaborgsomogyi!

**Review by maropu — APPROVED** [review]
Congrats!

**Comment by gaborgsomogyi** [comment]
Having some difficulty with my env but I'm on it to make my first merge :)

**Comment by viirya** [comment]
Congrats!

**Comment by huaxingao** [comment]
Congrats!

**Pull request closed without merging**